bitnami/kafka/values.yaml

```diff
@@ -1717,6 +1732,24 @@ zookeeper:
     accessModes:
       - ReadWriteOnce
     size: 8Gi
+  ## Service account for ZooKeeper to use.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+  ##
+  serviceAccount:
+    ## @param zookeeper.serviceAccount.create Enable creation of ServiceAccount for ZooKeeper pod
+    ##
+    create: false
+    ## @param zookeeper.serviceAccount.name The name of the ServiceAccount to use.
+    ## If not set and create is true, a name is generated using the common.names.fullname template
+    ##
+    name: ""
+    ## @param zookeeper.serviceAccount.automountServiceAccountToken Allows auto mount of ServiceAccountToken on the serviceAccount created
+    ## Can be set to false if pods using this serviceAccount do not need to use K8s API
+    ##
+    automountServiceAccountToken: true
+    ## @param zookeeper.serviceAccount.annotations Additional custom annotations for the ServiceAccount
+    ##
+    annotations: {}
```

Comment:
These are the default values so this shouldn't be necessary

https://github.com/bitnami/charts/blob/main/bitnami/zookeeper/values.yaml#L564

Reply:
Should I remove the params for README?